mobile/src/main/res/layout/mana_token.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ImageView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I think something needs to get changed here to have the image scale instead of clip when the completionview is a fixed size. Look into how android scales images. You may have to nest the ImageView in another layout.